PluginGenerator/Maven/MavenArtifactHandler.cs

```diff
@@ -0,0 +1,166 @@
+﻿//-----------------------------------------------------------------------
+// <copyright file="MavenArtifactHandler.cs" company="SonarSource SA and Microsoft Corporation">
+//   Copyright (c) SonarSource SA and Microsoft Corporation.  All rights reserved.
+//   Licensed under the MIT License. See License.txt in the project root for license information.
+// </copyright>
+//-----------------------------------------------------------------------
+using SonarQube.Plugins.Common;
+using System;
+using System.Diagnostics;
+using System.IO;
+using System.Net.Http;
+
+namespace SonarQube.Plugins.Maven
+{
+    public class MavenArtifactHandler : IMavenArtifactHandler
```

Comment:
Logically similar to the existing NuGet handler code: downloads artifacts from Maven Central and stores them in the specified location on disk.